main.go

```diff
@@ -82,6 +84,7 @@ and reports back results to the queue.
   Usage:
     generic-worker run                      [--config         CONFIG-FILE]
                                             [--configure-for-aws]
+                                            [--configure-for-gcp]
```

Comment:
I guess we could make these two options mutually exclusive, e.g.

```
[[--configure-for-aws]|[--configure-for-aws]]
```

I'm not too hot on docopt syntax - it might be subtly different. Not a huge gain to be had by doing this, but maybe solves some problems of non-determinism if both were passed in?

Reply:
This still needs doing.